src/lib.rs

```diff
@@ -21,6 +21,11 @@ pub struct PanCamPlugin;
 #[derive(Debug, Clone, Copy, SystemSet, PartialEq, Eq, Hash)]
 pub struct PanCamSystemSet;
 
+/// Event raised when an external trigger (i.e., something other than the player's input) might have
+/// invalidated the configured limits on camera bounds.
+#[derive(Event)]
+pub struct PanCamHintClampBoundsEvent;
```

Comment:
A bit unsure about the name. Maybe `InvalidatePanCamBoundsEvent`? Open for other suggetions. I think the documentation should also state that this is something you emit, not the library. Now it kind of sounds like we listen for external triggers, then emit events.